IDEAS/Fluid/BaseCircuits/PumpSupply_dp.mo

```diff
@@ -13,7 +13,8 @@ model PumpSupply_dp
     annotation(Dialog(tab="Dynamics", group="Filtered speed",enable=filteredSpeed));
 
   //Extensions
-  extends Interfaces.PartialPumpCircuit(redeclare Movers.FlowMachine_dp
```

Comment:
At this moment, `PumpSupply.mo` is no partial but `PumpSupply_m_flow.mo` with boolean on/off input instead of real signal input. Ideally, we should just keep `PumpSupply_m_flow.mo` and `PumpSupply_dp.mo` and indeed have a paremeter for boolean or real input, but it's no priority now.